src/components/checkboxes/_checkboxes.scss

```diff
@@ -139,7 +139,7 @@
     padding-left: $conditional-padding-left;
     border-left: $conditional-border-width solid $govuk-border-colour;
 
-    &[aria-hidden="true"] {
+    .js-enabled &.is-hidden {
```

Comment:
Should this class be called 'js-hidden'?

Could this be a common utility class?

https://github.com/alphagov/govuk-frontend/issues/815

Reply:
I think a common utility class would make sense.

(On a separate note, I think we should probably prefix our JS classes. But let's discuss that another time, I don't want to derail this PR 😁 )

Reply:
Yes I was thinking of `govuk-js-hidden`. I was even thinking of `js-hidden-govuk` as it's handy to recognise JS classes at a glance. But it just sounds too weird... 😅  

But `js-hidden` just seems like it would very easily conflict with something else.

Reply:
Makes sense, looks like we already use js-hidden, so maybe raise an issue for us to think about it after all 😬 

https://github.com/alphagov/govuk-frontend/search?q=js-hidden&unscoped_q=js-hidden

Reply:
I'm going to change the 'is-hidden' to 'js-hidden' to be consistent with tabs.